WhaleSpotting/react-app/src/components/SightingMapInfo.tsx

```diff
@@ -2,32 +2,64 @@ import React, { useEffect, useState } from "react";
 import WeatherApiModel from "../apiModels/WeatherApiModel";
 import { Chosen } from "./Map";
 import "../styles/SightingMapInfo.scss";
+import { Species, WhaleImageDictionary, WhaleVisualTextDictionary } from "../apiModels/CreateSightingApiModel";
 
 interface SightingMapInfoProps {
     chosen: Chosen | undefined;
 }
 
+interface IResponse {
+    lon: number | void,
+    lat: number | void,
+    species: Array<string>,
+}
+
 export default function SightingMapInfo({ chosen }: SightingMapInfoProps): JSX.Element {
 
     const key = process.env.REACT_APP_WEATHER_API_KEY;
 
-    const response = {
+    const [speciesData, setSpeciesData] = useState([]);
+
+    const response: IResponse = {
         lon: chosen?.lon,
         lat: chosen?.lat,
-        species: ["Whale", "Orca", "Dolphin"]
+        species: speciesData,
     };
 
     const [weatherData, setWeatherData] = useState<WeatherApiModel>();
     async function fetchWeather(): Promise<WeatherApiModel | void> {
```

Comment:
Can we move all the function definitions that don't need to be inside this function outside, and if they do need to be inside can we move them below the return please :)

Reply:
Addressed.